src/host_function.rs

```diff
@@ -80,7 +92,7 @@ mod tests {
     fn test_host_function() {
         let runtime = Runtime::builder()
             .use_system_allocator()
-            .register_host_function("extra", extra as *mut c_void)
+            .register_host_function("extra", extra as *mut c_void, CString::new("()i").unwrap())
```

Comment:
WAMR has changed register_host_function() signature in wasm_export.h recently. The 3rd arguments has been removed.